loadtest/loadtest.go

```diff
@@ -0,0 +1,328 @@
+package loadtest
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
+	"github.com/giantswarm/apprclient"
+	"github.com/giantswarm/backoff"
+	"github.com/giantswarm/e2e-harness/pkg/framework"
+	"github.com/giantswarm/helmclient"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/afero"
+	yaml "gopkg.in/yaml.v2"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/helm/pkg/helm"
+)
+
+type Config struct {
+	Clients        *Clients
```

Comment:
newbie question, why do you set those if I dont see they are used?

Reply:
I missed this when splitting the PR. In the next PR it will be used to create the Stormforger CLI job in the e2e cluster.